themes/ddbasic/template.php

```diff
@@ -767,13 +767,6 @@ function ddbasic_process_ting_object(&$vars) {
       break;
 
     case 'ting_object':
-
-      $uri_collection = entity_uri('ting_collection', $vars['object']);
-      $vars['ting_object_url_collection'] = url($uri_collection['path']);
-
-      $uri_object = entity_uri('ting_object', $vars['object']);
-      $vars['ting_object_url_object'] = url($uri_object['path']);
-
```

Comment:
@xendk Seems like there's still some references left to the $uri_object further down?

Reply:
FFS... Revived the variable.